support/support.rotation

```diff
@@ -30,4 +30,13 @@
 2021-05-31T01:00:00Z | aliok
 2021-06-07T01:00:00Z | zhongduo
 2021-06-14T01:00:00Z | devguyio
+2021-06-28T01:00:00Z | matzew
+2021-07-05T01:00:00Z | devguyio
```

Comment:
@aliok do you mind if we switch slots since I'll be off during that one?

Reply:
@devguyio sure no problem

Reply:
@lionelvillard wanna update and ship?

Reply:
done